errors/MutexError.js

```diff
@@ -0,0 +1,19 @@
+// Import base error
+const { BaseIsomerError } = require('./BaseError')
+
+class GetMutexError extends BaseIsomerError {
```

Comment:
For the backend, we decided to implement more general errors based on HTTP status codes because we ended up with too many custom errors when working on Homer. In general, we decided to follow Google's error response format ([link](https://cloud.google.com/storage/docs/json_api/v1/status-codes)) and status response codes ([link](https://developers.google.com/maps-booking/reference/rest-api-v3/status_codes))

Also, I feel like a situation where we are unable to obtain the mutex lock should be a `409` error

Reply:
Resolved in df09fce